pkgs/development/interpreters/racket/default.nix

```diff
@@ -60,6 +60,8 @@ stdenv.mkDerivation rec {
     for p in $(ls $out/bin/) ; do
       wrapProgram $out/bin/$p --set LD_LIBRARY_PATH "${LD_LIBRARY_PATH}";
     done
+    export PATH="$out/bin:$PATH";
```

Comment:
@henrytill I think so. I tried without the export, and it didn't work, but maybe I drew the wrong conclusion and the problem was elsewhere when I tried that. Is it a problem to give it the updated `PATH`? As far as I can see, it's not breaking purity. Could it cause other problems?